plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildAction.java

```diff
@@ -571,7 +571,8 @@ public double getTrend(final Baseline baseline, final Metric metric) {
     NavigableSet<Metric> getMetricsForSummary() {
         return new TreeSet<>(
                 Set.of(Metric.LINE, Metric.LOC, Metric.BRANCH, Metric.COMPLEXITY_DENSITY,
-                        Metric.MUTATION, Metric.TEST_STRENGTH, Metric.TESTS));
+                        Metric.MUTATION, Metric.TEST_STRENGTH, Metric.TESTS,
+                        Metric.MCDC_PAIR, Metric.METHOD, Metric.FUNCTION_CALL));
```

Comment:
The same here: existing users are normally not interested in method coverage. You will see all metrics when you open the tooltip. 

```suggestion
                        Metric.MCDC_PAIR, Metric.FUNCTION_CALL));
```

Reply:
Complete